packages/docs-app/src/styles/_nav.scss

```diff
@@ -32,20 +32,35 @@ $dark-package-colors: (
 
 .docs-logo {
   display: block;
+  opacity: 0.8;
   margin-right: $logo-margin;
   background-image: url("./assets/blueprint-logo.svg");
```

Comment:
remove these `background` properties, and the associated file.

while you're at it, refactor this entire class. i'll bet half the properties aren't necessary anymore.